data/helpers/usersDb.js

```diff
@@ -0,0 +1,42 @@
+const db = require('../../dbConfig')
+
+module.exports = {
+  getUsers,
+  getUser,
+  findUser,
+  addUser,
+  updateUser,
+  removeUser
+}
+
+function getUsers() {
+  return db('users')
+}
+function getUser(id) {
+  return db('users')
+    .where({ id })
+    .first()
+}
+function findUser(body) {
+  return db('users')
+    .where({ username: body.username })
+    .first()
+}
+function addUser(body) {
+  return db('users')
+    .insert(body)
+    .then(ids => ids[0])
```

Comment:
You could also pick the first element off of an array with [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Array_destructuring):
```javascript
  .then(([ id ]) => id)
```